src/main/resources/static/js/createSurvey.js

```diff
@@ -64,13 +86,33 @@ $("#createSurveyForm").on('click', '#addTextQuestion', function () {
      addNewQuestion(prompt, inputType)
 });
 
+// Button listener to add range question to evaluation to array for submission later
+$("#createSurveyForm").on('click', '#addRangeQuestion', function () {
+     prompt = $('#rangeQuestionPrompt').val()
+     inputType =  $(".inputSelect").val()
+	 min = $('#rangeQuestionMin').val()
+	 max = $('#rangeQuestionMax').val()
+	 if(isNaN(min) || isNaN(max)  ){
+		 if(max !> min){
```

Comment:
I don't think this operator exists.

Reply:
I'll fix that then.  Weirdly seemed to work in my testing